hunsri/EwMM-ProjectCS
# feat: powerup-weapons

## Description
- Add ammos to weapons as a part of powerup functionality.

## How to Test
- Go to `Powerups` scene
- New wave will start every 5 secs
- Waste your ammo before new wave starts
- Your ammo should be added on start of every wave
- Try deleting your `player.data` file and FFP2 masks should be removed from your weapons (default)

## Untested
- Additional ammos -> Not sure when to add additional (depends on `PlayerStats`)

### Scene: `Powerups`
